lib_xcore_math/api/xmath/xs3/vpu_info.h

```diff
@@ -78,13 +78,13 @@ enum {
     VPU_INT32_MAX =  0x7FFFFFFF,
     /** The lower saturation bound for 32-bit elements and 32-bit accumulators */
     VPU_INT32_MIN = -0x7FFFFFFF,
-
-    /** The upper saturation bound for 40-bit accumulators */
-    VPU_INT40_MAX =  0x7FFFFFFFFFLL,
-    /** The lower saturation bound for 40-bit accumulators */
-    VPU_INT40_MIN = -0x7FFFFFFFFFLL,
 };
 
+/** The upper saturation bound for 40-bit accumulators */
+#define VPU_INT40_MAX  0x7FFFFFFFFFLL
+/** The lower saturation bound for 40-bit accumulators */
+#define VPU_INT40_MIN -0x7FFFFFFFFFLL
+
```

Comment:
I'm not sure why this has been moved.

Reply:
> I'm not sure why this has been moved.

Enums items for Windows are 4 bytes long, so I can't use long long values.

Reply:
I used an ifdef to distinguish the Windows implementation from the others.